python/tests/test_tools/test_base/test_runtime.py

```diff
@@ -151,3 +153,25 @@ def inverse(
     assert len(actions) == 2
     assert square in actions
     assert inverse in actions
+
+
+def test_untyped_param() -> None:
+    """Test if error is raised if a param is missing type annotation."""
+    with pytest.raises(
+        InvalidRuntimeAction,
+        match=re.escape(
+            "Following arguments are missing type annotations: {'execute_request'}"
+        ),
+    ):
+
+        # pylint: disable=unused-argument
+        @action(toolname="github")
+        def list_prs(owner: str, repo: str, execute_request) -> list[str]:
```

Comment:
The docstring is missing a description for the `execute_request` parameter. Even though this is a test case demonstrating the error, it would be helpful to document all parameters consistently.